### DISCUSSION TITLE
Gradient

### DISCUSSION THREAD

**Discussion by cocobom** [skia4delphi/skia4delphi#257] [General]
Enable skia in FMX and set the fill property to gradient color,  run it , gradient color cannot be presented, any idea? 11.3 and skia4d 5.0

**Comment by viniciusfbb** [comment]
Hi @cocobom, it is a know issue of v5.0.0, but fixed on v6 beta. Please, use our last version **[v6.0.0-beta3](https://github.com/skia4delphi/skia4delphi/releases)**

**Reply by cocobom** [reply]
ok, trying...
